embrace-android-delivery/build.gradle.kts

```diff
@@ -13,4 +13,10 @@ apiValidation.validationDisabled = true
 dependencies {
     implementation(project(":embrace-android-payload"))
     implementation(project(":embrace-android-core"))
+    compileOnly(platform(libs.opentelemetry.bom))
```

Comment:
So the implementation of the OTel clock that we use effectively points to the internal implementation of our own clock. I was pretty loosey goosey with which clock I reference because everything was in one module, but if it matters, we can clean that up so we don't need OTel in the module.

Perhaps one of us can do this after this is mreged?

Reply:
I'll clean this up after merging. It looks like we only need the class because `FakeInitModule` is referencing it so it's just an issue with our tests